mbox-operator/roles/mbox/meta/main.yml

```
@@ -0,0 +1,62 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```

```suggestion
  author: CPE Team
  description: MBox operator
  company: Red Hat
```

We can change the description to something more fitting later.